cmp/report_compare.go

```diff
@@ -89,13 +89,13 @@ func (opts formatOptions) FormatDiff(v *valueNode) textNode {
 			if v.NumDiff == 0 {
 				outx := opts.FormatValue(v.ValueX, visitedPointers{})
 				outy := opts.FormatValue(v.ValueY, visitedPointers{})
-				if v.NumIgnored > 0 && v.NumSame == 0 {
+				switch {
+				case v.NumIgnored > 0 && v.NumSame == 0:
 					return textEllipsis
-				} else if outx.Len() < outy.Len() {
+				case outx.Len() < outy.Len():
```

Comment:
This is deliberate.

Both branches have equal probability of being executed. Keeping them on the same indent highlights this fact.